src/Spectre/Controllers/PreparationsController.cs

```diff
@@ -74,5 +77,43 @@ public Spectrum Get(int id, int spectrumId)
 
             return new Spectrum() { Id = spectrumId, Intensities = intensities, Mz = mz, X = coordinates.X, Y = coordinates.Y };
         }
+
+        /// <summary>
+        /// Gets single heatmap of a specified preparation based on provided mz.
+        /// </summary>
+        /// <param name="id">Preparation identifier.</param>
+        /// <param name="mz">Mz value serving as heatmap basis.</param>
+        /// <returns>Heatmap</returns>
+        /// <exception cref="ArgumentException">Thrown when provided mz is lower 
+        /// than zero, or is invalid for a given dataset</exception>
+        public Heatmap Get(int id, double mz)
```

Comment:
I think we can leave it for now, but I'm wondering on splitting this controller at some point to provide these functions accepting only IDs. Doubles are hard to compare.